lib/bencher_adapter/tool_output/rust/iai_callgrind/global-bus-events.txt

```diff
@@ -0,0 +1,21 @@
+
```

Comment:
I think `iai_callgrind` also supports many other events that can be optionally turned on/off using other flags. I wonder if it is worth it adding all the possible combinations of them?
But happy to do so if you like.

Reply:
> I wonder if it is worth it adding all the possible combinations of them?

Yes, I think it would be worth adding them. Let's just keep this PR exclusive to Ge. If you could do some digging, come up with a list of all options, and create a follow on issue. Depending on how many other options there are will dictate how I want to proceed (chunking or one at a time).

I really appreciate your help!

Reply:
Sure. Added the test you suggested.
I'm not sure about other events yet, since we don't use them/I have not seen them before. They are also coming from multiple projects/tools that are invoked via Valgrind. So not all are used/mentioned in the `iai-callgrind` codebase. Maybe depending on future bug reports/user requests?